packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -375,10 +375,22 @@ data:
 function isReady {
   geth attach << EOF
 
+    // Deployment configuration
+    var maxpeers = {{ .maxpeers | default 1150 }}
+    var lightpeers = {{ .light_maxpeers | default 1000 }}
```

Comment:
I'm thinking if we set the defaults here then this should continue to work even if `.maxpeers` or `.light_maxpeers` has not been set. But that sounds like it could cause problems, for example if `.maxpeers` or `.light_maxpeers` were renamed then we'd actually want to update this code to match. So if possible I think it would be better to fail here if `.maxpeers` or `.light_maxpeers` have not been defined.

Reply:
I do not see a potential problem if those variables are renamed. `.maxpeers` and `.light_maxpeers` are variables only used in the `"common.node-health-check"` context, so I guess if they are eventually renamed, they should be updated in the subtemplate context. Also if these default values used for the healtcheck does not work for the node configuration, they upgrade would probably stop with the first pod upgraded (as Kubernetes will wait to the pod to be `ready`), so in that scenario that could call the attention of the operator (although if node is synced the block age check will pass with independence of number of peers)

My rationale to include default values in this subtemplate is that if someone is using the `common` chart outside monorepo, as we are not currently versioning the charts, the change can break deployed versions and require a change there.